_data/mentors.yml

```diff
@@ -181,6 +181,10 @@
   github: ananyo2012
   image: MaitiAnanyo.jpg
   twitter: MaitiAnanyo
+  
+- name: Pawan Pal
+  github: Pawan Pal
```

Comment:
Here's where you're wrong: github usernames do not contain spaces. Find his twitter as well. It's not hard to find. Google it.